Examples/PullServerDeployementVerificationTest/PullServerSetupTests.ps1

```diff
@@ -0,0 +1,135 @@
+﻿
```

Comment:
Tests should be located in the tests directory, we should invoke configuration with DscWebService resource to set up PullServer and then execute these to validate.